libs/langchain/tests/integration_tests/vectorstores/test_cratedb.py

```diff
@@ -265,10 +261,9 @@ def test_cratedb_with_filter_distant_match() -> None:
         connection_string=CONNECTION_STRING,
         pre_delete_collection=True,
     )
+    output = docsearch.similarity_search_with_score("foo", k=2, filter={"page": "2"})
```

Comment:
why is `k` changed here?

Reply:
My intention was probably to improve test stability, by fiddling with parameters. Apologies, the outcome is non-deterministic, see above.